formation/for_requests.py

```diff
@@ -100,40 +105,39 @@ def raw_response(ctx):
 def json_response(ctx):
     res = get_response(ctx)
     if not res:
-        return (None, None, None)
-    return (res.json(), res.status_code, res.headers)
+        return None, None, None
+    return res.json(), res.status_code, res.headers
 
 
 @staticmethod
 def xmltodict_response(ctx):
     res = get_response(ctx)
     if not res:
-        return (None, None, None)
-    return (xmltodict.parse(res.text), res.status_code, res.headers)
+        return None, None, None
+    return xmltodict.parse(res.text), res.status_code, res.headers
 
 
 @staticmethod
 def html_response(ctx):
     res = get_response(ctx)
     if not res:
-        return (None, None, None)
-    return (html.fromstring(res.content), res.status_code, res.headers)
+        return None, None, None
+    return html.fromstring(res.content), res.status_code, res.headers
 
 
 @staticmethod
 def text_response(ctx):
     res = get_response(ctx)
     if not res:
-        return (None, None, None)
-    return (res.text, res.status_code, res.headers)
+        return None, None, None
+    return res.text, res.status_code, res.headers
 
 
 def build_sender(middleware=[], base_uri=None, response_as=None):
     wrapped = wrap(requests_adapter, middleware=middleware)
 
     def sender(method, url, session_context={}, params={}, **kwargs):
-        resolved_response_as = kwargs.get("response_as", response_as) or _raw_response
-        print(resolved_response_as)
+        resolved_response_as = kwargs.pop("response_as", response_as) or raw_response
```

Comment:
you mean `or _raw_response`?

Reply:
Also not sure about pop/get. Given a nonexisting key, behaves similarly:
```
>>> d = { 'a':1 }
>>> d.get('b', 2)
2
``

Reply:
yeah but `pop` will remove it from the dict

Reply:
if `response_as` gets to [`FormationHttpRequest`](https://github.com/jondot/formation/blob/1d100b55414e781e2dc8b49497a98f3484daa35b/formation/for_requests.py#L144) in the `kwargs` it will explode - `attrs` won't accept it

Reply:
ah! now i understand
this is part of a major attrs upgrade, good catch.
since pop is side effect (I usually stay away from `pop`/`delete` and friends) -- if we put response as in the method signature would that be the same?

```py
    def sender(method, url, session_context={}, params={}, response_as=_raw_response, **kwargs):
```

Reply:
right. Had to change the outer `response_as` so it won't be shadowed

Reply:
Why is raw_response not exported? Am I missing anything? Or that can be obtained without specifying raw_response class attr?